pixeltable/catalog/table_version.py

```diff
@@ -6,16 +6,17 @@
 import logging
 import time
 import uuid
-from typing import TYPE_CHECKING, Any, Iterable, Iterator, Literal, Optional, Tuple
+from typing import TYPE_CHECKING, Any, Callable, Iterable, Iterator, Literal, Optional, Tuple
 from uuid import UUID
 
 import jsonschema.exceptions
 import sqlalchemy as sql
 from sqlalchemy import exc as sql_exc
 
-import pixeltable as pxt
 import pixeltable.exceptions as excs
-from pixeltable import exprs, index
+import pixeltable.exprs as exprs
```

Comment:
Better to condense these four lines into:
```
from pixeltable import exceptions as excs, exprs, index, type_system as ts
```
That would mirror the pattern used throughout the codebase